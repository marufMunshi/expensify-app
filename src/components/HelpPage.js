import React from 'react';

const HelpPage = () => {
    return (
        <div>
            This from help component.
        </div>
    );
};

export default HelpPage;