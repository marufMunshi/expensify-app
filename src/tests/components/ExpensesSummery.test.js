import React from 'react';
import { shallow } from 'enzyme';
import numeral from 'numeral';
import { ExpensesSummery } from '../../components/ExpensesSummery';
import selectExpensesTotal from '../../selectors/expenses-total';
import expenses from '../fixtures/expenses';

let expensesTotal, expenseCount;

expensesTotal = selectExpensesTotal(expenses);
expenseCount = expenses.length;

test('should render expensesTotal and expenseCount correctly', () => {
    const wrapper = shallow( 
        <ExpensesSummery 
            expenseTotal={expensesTotal}
            expenseCount={expenseCount}              
        /> 
    );
    expect(wrapper).toMatchSnapshot();
});
