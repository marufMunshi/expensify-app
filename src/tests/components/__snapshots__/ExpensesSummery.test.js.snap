// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensesTotal and expenseCount correctly 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totalling 
    $1,141.95
  </h1>
</div>
`;
